### ISSUE TITLE
Accesibility issue : Right navigator button > is not working with tab  in Library-> Resources-> reading

### ISSUE THREAD

**Issue by Pursottam6003** [learningequality/kolibri#10371]
## Observed behavior

Just exploring the Kolibri and got some issue that needs to be reviewed 

**The read section of the Kolibri Resources section is not working with tabs properly and its really important for a website to fully functioned with tabs**

It improves accessibility and anyone can use it with a single button even a handicapped person using tabs

[Screencast from 02-04-23 01:25:35 AM IST.webm](https://user-images.githubusercontent.com/81693090/229313508-4e40ca4c-4ba7-44b6-b58b-20acefdd75d1.webm)

## Expected behavior

Similar to left navigation  < it should also navigate to the right navigation  > button but it does not happen

## User-facing consequences

…

## Steps to reproduce

1. Go to learn section
2. Navigate toward the resource section
3. Open any chapter and try to read 
4. Now use tabs to get the navigation button < of left 
5. You will observe its not working for right 

## Context

I would like to work on it if I get some guidance from mentors

**Comment by shivansh-bhatnagar18** [comment]
@MisRob Please Assign me this issue. I would love to work on this

**Comment by AlexVelezLl** [comment]
Hi @shivansh-bhatnagar18! Thanks for your interest in contributing to Kolibri! :) I will assign this issue to you :open_hands:. Let us know if you have any question.

**Comment by kafukoM** [comment]
Any updates on this issue? Would like to help with resolving this?

**Comment by MisRob** [comment]
Hi @shivansh-bhatnagar18, are you working on this or can we re-assign to @kafukoM?

**Comment by shivansh-bhatnagar18** [comment]
Was distracted for a while. Will be continuing working on the issue and will fix it asap

**Comment by MisRob** [comment]
@shivansh-bhatnagar18 no pressure, just didn't want to re-assign before checking on it.
@kafukoM it'd be best to find another "help wanted" issue for now.

Thanks both.

**Comment by shivansh-bhatnagar18** [comment]
@MisRob, I guess the issue has already been resolved and the issue should be marked as completed or is it something that I am missing. I am attaching a video for the same
[Screencast from 2024-03-22 01-00-17.webm](https://github.com/learningequality/kolibri/assets/117429248/dfa7c4b4-d5c5-4981-bb99-89f9a7c35a93)

**Comment by LianaHarris360** [comment]
Hi @shivansh-bhatnagar18, this problem is not yet resolved. It seems to be occurring with resources similar to the one shown in the screenshot below. 

![pdf](https://github.com/learningequality/kolibri/assets/46411498/93db07ed-7b1b-419e-993c-552ca68e1c9b)

**Comment by shivansh-bhatnagar18** [comment]
Ok @LianaHarris360 , thanks for pointing this out. Will look into it

**Comment by MisRob** [comment]
Hi @shivansh-bhatnagar18, are you working on this or would it be better to unassign?

**Comment by shivansh-bhatnagar18** [comment]
@MisRob I apologize for the delay. I have unassigned the issue from myself and will look forward to contribute in other ways

**Comment by nathanaelg16** [comment]
Hi team, I have a fix for this issue, please see my PR  #12183

**Comment by AlexVelezLl** [comment]
Hi @nathanaelg16 :wave:. Thanks for your interest in contributing to Kolibri :rocket:. We will take a look at the PR.
But just a quick note, it is recommended to first ask us to assign you an issue before continuing with it =). Just to make sure that no one else is looking at the same issue. You can read more about our contributing guidelines [here](https://github.com/learningequality/kolibri/blob/develop/CONTRIBUTING.md) :open_hands:.

**Comment by MisRob** [comment]
Needs some clarification, more here https://github.com/learningequality/kolibri/pull/12183

**Comment by MisRob** [comment]
@nathanaelg16 I'm sorry but we are not sure when we can get to clarify exact expectations you and @radinamatic started discussing in https://github.com/learningequality/kolibri/pull/12183. So I will unassign this issue now and remove it from the issues open for contribution. Thank you for your time on it, it is still valuable since it surfaced the gaps.
